library/lua/script-manager.lua

```diff
@@ -57,4 +60,107 @@ function list()
     end
 end
 
+---------------------
+-- mod script paths
+
+-- this perhaps could/should be queried from the Steam API
+-- are there any installation configurations where this will be wrong, though?
```

Comment:
steam (on windows at least) allows for you to install individual games on a separate drive. i don't know whether, if you do this, the workshop for any given game will still be on the same drive as the game, or if there is a single central workshop off the main steam installation folder in such situations. if the latter is the case, this approach will fail for someone who installs DF on a different drive than their main steam installation

Reply:
I'll open this as a followup issue. We'd need a Steam library wrapper like our DFSDL module, I think, which is something we can add later.

Reply:
It also *may* not be necessary to read from the steam workshop directory at all. I noticed while testing that mods appeared to be copied from the steam workshop directory into the df mods/ directory on startup, so **maybe** just reading the mods/ directory is good enough. I'll have to test this more when I have more time, though.